mbuild/tests/test_utils.py

```diff
@@ -9,3 +11,8 @@ def test_assert_port_exists(self, ch2):
         assert_port_exists('up', ch2)
         with pytest.raises(ValueError):
             assert_port_exists('dog', ch2)
+
+    def test_structure_reproducibility(self, alkane_monolayer):
+        filename = 'monolayer-tmp.pdb'
+        alkane_monolayer.save(filename)
+        assert filecmp.cmp(get_fn('monolayer.pdb'), filename)
```

Comment:
Excellent regression test!

Reply:
Thanks. Ideally we would also be checking to make sure the topology is reproduced as well (not as relevant for this PR), but it seems the ordering of bonds, angles, etc. is lost in ParmEd so a simple file comparison won't work.

Reply:
Actually it looks like the `filecmp` comparison won't work when comparing files generated on Mac or Linus with those on Windows (which is why AppVeyor is failing). @justinGilmer has suggested using `difflib` which looks like it should do the trick, so I'll update the test tonight.

Reply:
Something that I've wanted to do for a while is write a proper [__eq__ function for compound](https://github.com/mosdef-hub/mbuild/issues/64) which would let you compare the actual objects created in memory rather than the .pdb file. Clearly I've been pushing that off for quite some time based on the reassigned milestones 😬 

Not sure if that's necessarily better for this particular test but would be useful in general.